shenyu-plugin/shenyu-plugin-response/src/main/java/org/apache/shenyu/plugin/response/strategy/RPCMessageWriter.java

```diff
@@ -38,11 +37,11 @@
         return chain.execute(exchange).then(Mono.defer(() -> {
             Object result = exchange.getAttribute(Constants.RPC_RESULT);
             if (Objects.isNull(result)) {
-                Object error = ShenyuResultWrap.error(ShenyuResultEnum.SERVICE_RESULT_ERROR.getCode(), ShenyuResultEnum.SERVICE_RESULT_ERROR.getMsg(), null);
+                Object error = ShenyuResultWrap.error(ShenyuResultEnum.SERVICE_RESULT_ERROR.getCode(),
+                        ShenyuResultEnum.SERVICE_RESULT_ERROR.getMsg(), null);
                 return WebFluxResultUtils.result(exchange, error);
             }
-            Object success = ShenyuResultWrap.success(ShenyuResultEnum.SUCCESS.getCode(), ShenyuResultEnum.SUCCESS.getMsg(), JsonUtils.removeClass(result));
-            return WebFluxResultUtils.result(exchange, success);
+            return WebFluxResultUtils.result(exchange, result);
```

Comment:
Not sure is the right way.
Does it equal?
```
Object result = exchange.getAttribute(Constants.RPC_RESULT);
WebFluxResultUtils.result(exchange, result); 
```
and 
```
Object success = ShenyuResultWrap.success(ShenyuResultEnum.SUCCESS.getCode(), ShenyuResultEnum.SUCCESS.getMsg(), JsonUtils.removeClass(result));
WebFluxResultUtils.result(exchange, result);
```

Reply:
Before the discussion, the RPC result uses the upstream result and does not wrap with code, message, and data, like HTTP call.